File: src/main/java/net/igneo/icv/entity/helmet/glacialImpasse/iceSpikeSpawner/IceSpikeSpawnerRenderer.java
package net.igneo.icv.entity.helmet.glacialImpasse.iceSpikeSpawner;

import net.minecraft.client.renderer.entity.EntityRendererProvider;
import software.bernie.geckolib.renderer.GeoEntityRenderer;

public class IceSpikeSpawnerRenderer extends GeoEntityRenderer<IceSpikeSpawnerEntity> {
    public IceSpikeSpawnerRenderer(EntityRendererProvider.Context renderManager) {
        super(renderManager, new iceSpikeSpawnerModel());
    }
}

File: src/main/java/net/igneo/icv/entity/helmet/glacialImpasse/iceSpikeSpawner/IceSpikeSpawnerRenderer.kt
package net.igneo.icv.entity.helmet.glacialImpasse.iceSpikeSpawner

import net.minecraft.client.renderer.entity.EntityRendererProvider
import software.bernie.geckolib.renderer.GeoEntityRenderer

class IceSpikeSpawnerRenderer(renderManager: EntityRendererProvider.Context) :
    GeoEntityRenderer<IceSpikeSpawnerEntity?>(renderManager, iceSpikeSpawnerModel())
